arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
+// Project: http://resources.arcgis.com/en/help/arcgis-rest-api/
+// Definitions by: Jeff Jacobson <https://github.com/JeffJacobson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace ArcGis.Rest {
```

Comment:
Could this be a module instead of a namespace? Even if a `argis-rest-api` package doesn't really exist, TypeScript will elide imports if every import is a type, so it won't cause errors at runtime.